api/src/services/taxonomy-service.ts

```diff
@@ -73,9 +72,10 @@ export class TaxonomyService {
    * @memberof TaxonomyService
    */
   async addItisTaxonRecord(itisSolrResponse: ItisSolrSearchResponse): Promise<TaxonRecord> {
-    let commonName = null;
+    let commonNames = null;
     if (itisSolrResponse.commonNames) {
-      commonName = itisSolrResponse.commonNames[0].split('$')[1];
+      const firstEnglishName = itisSolrResponse.commonNames.find((name) => name.split('$')[2] === 'English');
+      commonNames = firstEnglishName ? firstEnglishName.split('$')[1] : null;
```

Comment:
I would expect `commonNames` to be an array. I think leaving `commonNames` as type `string | null` may lead to some confusion. (See my other comment I left in taxonomy-repository)

Reply:
Thanks. commonNames is now always an array.